sha256.js

```diff
@@ -53,8 +50,6 @@ Sha256.prototype.init = function () {
   this._g = 0x1f83d9ab|0
   this._h = 0x5be0cd19|0
 
-  this._len = this._s = 0
```

Comment:
initialized in `hash.js`, no need to do it here.  Doesn't even get used in this part of the code.